frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/EnvironmentVariantsModal.tsx

```diff
@@ -225,7 +227,16 @@ export const EnvironmentVariantsModal = ({
         isChangeRequestConfigured(environment?.name || '') &&
         uiConfig.flags.crOnVariants;
 
-    const stickiness = variants[0]?.stickiness || 'default';
+    const projectStickiness = localStorage.getItem(
+        `defaultStickiness.${projectId}`
+    );
+
+    const defaultStickiness =
+        Boolean(projectScopedStickiness) && projectStickiness != null
+            ? projectStickiness
+            : 'default';
+
+    const stickiness = variants[0]?.stickiness || defaultStickiness;
```

Comment:
This is low level details of the stickiness leaking out to the client code. As a cosumer of stickiness code I'd expect a hook useDefaultProjectStickiness() that spares me the details how it's calculated and that it's using local storage. It will be much easier to evolve without touching lots of code.

Reply:
makes sense - will implement